src/cast_core/src/RollingCollections.cpp

```diff
@@ -62,8 +62,8 @@ static std::string getRollingCollectionName() {
     // The id is tracked globally and increments for every collection created.
     static std::atomic_long id = 0;
     std::stringstream ss;
-    ss << "r"
-       << "_" << id << "_" << getMillisecondsSinceEpoch();
+    // Create a unique collection name that sorts lexographically by time.
+    ss << "r_" << getMillisecondsSinceEpoch() << "_" << id;
```

Comment:
The intention is to then merge this after the OplogTailer fix yeah?

Reply:
That would be easiest, but the other order would work as long as the changes to this file get merged correctly into the OplogTailer branch.